entities/accounts/edit/general-information/general-information.tsx

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 
-import { AddProfilePhoto } from '@/features/accounts/edit';
+import { AddProfilePhoto, SaveChangesButton } from '@/features/accounts/edit';
 import { ProfileForm } from '@/features/accounts/edit/profile-form/profile-form';
 
 import style from './general-information.module.scss';
 
 export const GeneralInformation = () => {
   return (
-    <div className={style.container}>
-      <AddProfilePhoto />
+    <>
+      <div className={style.container}>
+        <AddProfilePhoto />
 
-      <ProfileForm />
-    </div>
+        <ProfileForm />
+      </div>
+
+      <SaveChangesButton />
```

Comment:
There is no benefit to separating the button from `ProfileForm`. It does not improve readability by reducing the form code (a single button!), nor does it simplify the logic (there is no logic in the `SaveChangesButton` code). The button should be placed next to other form elements.

Reply:
This was done due to the page layout. The button is separated from the form by a separator. Then we need to change styles

Reply:
The `Submit` button will not be shared between multiple pages in the layout provided by the tabs at the top of the page. It belongs to the `ProfileForm` only. You've ripped it away from the `ProfileForm` and made it part of `GeneralInformation`. Does the form need a footer? And is it necessary to detach the submit button from the form to simulate a footer? I think these are rhetorical questions.

Even if you prefer to wrap the form input to create a separator using the `border-bottom` CSS property, there is still no reason to make the button a separate component. You can place it next to the form wrapper. The `ProfileForm` is all just a form. 

By the way, you can simply place a `div` with `aria-role="separator"` before the last `Textfield`. This will eliminate the wrapper altogether. If you think this is a weird solution, take a look at a [similar solution](https://www.radix-ui.com/primitives/docs/components/separator#accessibility) from the Radix-UI library.
Also, the button doesn't need a container with `display: flex` properties in principle. It is enough to give the button the following CSS: 
```css
margin-top: 24px;
margin-left: auto;
display: block;
```

Reply:
By the way, the `AddPforilePhoto` component is also part of the form. And it should be placed there as well. Or it can be wrapped by the `ProfileForm` component by calling `useFormContenxt` inside.  Then you `ProfileForm` should add `<Form Provider />`. Since `AddPforilePhoto` is also involved in generating user information and its data should also be reflected in the `onSubmit` callback.

If everything is included in `ProfileForm` as I suggest, then it turns out that the `GeneralInformation` component is not needed? That's right! In fact, we should include all fields related to the form in it. Then we need only one component, whether it is called `GeneralInformation` or `ProfileForm`. The tabs surrounding it are part of the Layout component assigned to the `getLayout` property of each page corresponding to the tab on the tab. And of course, tabs must then perform page navigation.

Reply:
Why you decide `AddPforilePhoto` component is part of the form? It works separate from `ProfileForm`, it has its own Save button (on modal window), it has its own upload photo request, etc. And according our API, we have next fields for PUT request. 
```
{
  "username": "John-Doe",
  "firstname": "John",
  "lastname": "Doe",
  "dateOfBirth": "01.01.2023",
  "aboutMe": "any text",
  "city": "Minsk",
  "country": "Belarus"
}
```
When we click `Save Changes` we should send information above. But when we need to get info from the back, we get all these plus profile photo.